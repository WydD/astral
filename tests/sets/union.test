<?xml version="1.0" encoding="UTF-8"?>
<astral xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://astral.ligforge.imag.fr/schema/query.xsd">
<query t0="auto">
<union>
<source attributes="id,T,PHYID,temp" id="R1" sourceAttributes="id,T,PHYID,temp" type="Relation"/>
<source attributes="id,T,PHYID,temp" id="R2" sourceAttributes="id,T,PHYID,temp" type="Relation"/>
</union>
</query>
<sources>
<source attributes="id,T,PHYID,temp" file="tests/sets/R1.R" id="R1" impl="FileRelationImpl" type="Relation"/>
<source attributes="id,T,PHYID,temp" file="tests/sets/R2.R" id="R2" impl="FileRelationImpl" type="Relation"/>
</sources>
</astral>
