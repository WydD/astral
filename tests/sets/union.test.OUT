id,temp,T
t=1000
1,27,1000
1,27,1000
1,29,1000
2,27,1000
2,27,1000
2,29,1000
t=1005
1,33,1005
2,33,1005
t=1010
1,25,1010
1,31,1010
1,33,1010
2,25,1010
2,31,1010
2,33,1010
t=1015
1,31,1015
2,31,1015
t=1020
t=1025
1,33,1025
1,29,1025
2,33,1025
2,29,1025
